common/ext/class.Extension.php

```diff
@@ -497,12 +497,13 @@ public function getConfigHeader($key)
     /**
      * @throws common_ext_ManifestException
      * @throws common_ext_ManifestNotFoundException
+     * @throws common_ext_UpdaterNotFoundException
```

Comment:
I don't see a definition of the exception.
Have you forgotten to commit/push it?

Reply:
indeed